sleap/gui/dialogs/formbuilder.py

```diff
@@ -475,14 +484,12 @@ def add_item(self, item: Dict[Text, Any]):
             if "range" in item.keys():
                 min, max = list(map(float, item["range"].split(",")))
             field.setRange(min, max)
-                
 
             field.setValue(item["default"])
 
             field.valueChanged.connect(lambda: self.valueChanged.emit())
 
-        # int: show spinbox (number w/ up/down controls)
-        elif item["type"] == "int":
+        elif item["type"] == "int":  # int: show spinbox (number w/ up/down controls)
```

Comment:
I liked these better as block comments. Why? 
1. They're easier to see/read
2. The comment explains the block of code to follow not just the single line

```suggestion
        # int: show spinbox (number w/ up/down controls)
        elif item["type"] == "int":
```